gpumon/gpumon_server.ml

```diff
@@ -157,14 +159,16 @@ module Make (I : Interface) : IMPLEMENTATION = struct
                  | Nvml.Other ->
                      Some Gpumon_interface.Other
                  | Nvml.None ->
-                     None)
-            |> Xapi_stdext_std.Listext.List.setify
+                     None
```

Comment:
What is happening here? Why do we need the `setify` no longer?

Reply:
Uniquifying the list is done only when it's necessary later on, with `List.sort_uniq`